src/libhirte/ini/config.c

```diff
@@ -5,21 +5,33 @@
 #include "ini.h"
 #include "libhirte/common/common.h"
 
-int match(const char *section, const char *check_section, const char *name, const char *check_name) {
-        if (streq(section, check_section) == 0 && streq(name, check_name) == 0) {
```

Comment:
Just a note, we have been using streq() instead of strcmp() according to systemd coding style guide
https://github.com/systemd/systemd/blob/main/docs/CODING_STYLE.md#functions-to-avoid

Reply:
Its all fine to use streq if you want to compare two strings for equality, but these are callbacks from the hashtable implementation, and they are supposed to return 0 on match, so streq would return the wrong value.

Reply:
Ack